src/peergos/server/tests/MultiUserTests.java

```diff
@@ -490,6 +490,24 @@ public void followPeergos() throws Exception {
 
         List<FollowRequestWithCipherText> u1Requests = u1.processFollowRequests().get();
         assertTrue("Receive a follow request", u1Requests.size() > 0);
+
+        // Check that user w can send feedback to the user peergos.
+        String feedback = "Here's some constructive feedback!";
+        CompletableFuture<Boolean> testFeedbackSubmission = u2.submitFeedback(feedback);
+        assertTrue("Feedback submission was successful!", testFeedbackSubmission.get() == true);
+
+        // Can peergos read the feedback file?
+        Path filePath = Paths.get(u1.username, "feedback");
+        Optional<FileWrapper> feedbackFile = u1.getByPath(filePath).get();
```

Comment:
this is the feedback dir, not the file. You need to call getChildren on this to get the file

Reply:
Got it. Thanks 👍